builder/qemu/step_prepare_efivars.go

```diff
@@ -0,0 +1,71 @@
+package qemu
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+)
+
+// stepPrepareEfivars copies the EFIVars file to the output, so we can boot
+// and use it as a RW flash drive
+type stepPrepareEfivars struct {
+	EFIEnabled  bool
+	OutputDir   string
+	EFIVarsPath string
+}
+
+const efivarStateKey string = "EFI_VARS_FILE_PATH"
+
+func (prepEfi *stepPrepareEfivars) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
```

Comment:
For step receivers we generally go with the receiver name of `s` across plugins. It is a common nomenclature that helps reduce the mental model of working within step code . Calling it prepEfi confused me because I thought it was different from the step configuration receiver. 

```suggestion
func (s *stepPrepareEfivars) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
```